### DISCUSSION TITLE
mi telefono es un iphone 12 rojo appled por favor cerrar mi informcion de ese androi

### DISCUSSION THREAD

**Discussion by 9122576424** [community/community#111250] [Mobile]
### Select Topic Area

Show & Tell

### Body

no conosco ese androi

**Comment by queenofcorgis** [comment]
Hi @9122576424 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
